cineast-api/src/main/java/org/vitrivr/cineast/api/messages/result/TemporalQueryResult.java

```diff
@@ -0,0 +1,28 @@
+package org.vitrivr.cineast.api.messages.result;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.List;
+import org.vitrivr.cineast.api.messages.abstracts.AbstractQueryResultMessage;
+import org.vitrivr.cineast.api.messages.interfaces.MessageType;
+import org.vitrivr.cineast.core.data.TemporalObject;
+
+public class TemporalQueryResult extends AbstractQueryResultMessage<TemporalObject> {
+
+  @JsonCreator
+  public TemporalQueryResult(String queryId, List<TemporalObject> content) {
+    super(queryId, TemporalObject.class, content);
+  }
+
+  @Override
+  public MessageType getMessageType() {
+    return MessageType.QR_TEMPORAL;
+  }
+
+  @Override
+  public String toString() {
+    return "TemporalQueryResult{" +
```

Comment:
Ideally, you'd use `return ReflectionToStringBuilder.toString(this, ToStringStyle.JSON_STYLE);`, this makes future changes easier and also prints the whole thing

Reply:
As it isn't really necessary to even have this functionality I would remove it entirely.

Reply:
I like having meaningful toString() methods for API classes since it improves logging

Reply:
Me too, so I will open a PR too sometime soon to add those to all API classes as most are either missing one altogether or do not utilize `return ReflectionToStringBuilder.toString(this, ToStringStyle.JSON_STYLE);`.